docs/content/new-tutorial/intro-to-assets.mdx

```diff
@@ -0,0 +1,83 @@
+
+---
+
+## Introduction to Software-defined assets
+
+Before building a Dagster project, you'll first learn about Dagster's core concept: the Software-defined asset (SDA).
+
+An asset is an object in persistent storage that captures some understanding of the world. Assets can be any type of object, such as:
+
+- A database table or view
+- A file, such as in your local machine or blob storage like Amazon S3
+- A machine learning model
+
+A Software-defined asset is Dagster's concept that couples an asset to the function and upstream assets that are used to produce its contents.
+
+If you have an existing data pipeline, you likely already have assets.
+
+### Goals
+
+By the end of this section, you will:
+
+- Know what assets are
+- Know the power of Software-defined assets
+- Know when to use task-based orchestration with assets
+- Start to think about what parts of your current pipeline are assets
+
+### Parts of a Software-defined asset
+
+Software-defined assets wrap objects to enable data engineers to do more with them.
+
+As a Dagster user, you're able to build assets in code. In the sample asset below, the data frame returned is an asset and Dagster recognizes that the function is an SDA 
+
+```
+@asset
+def hackernews_topstories(hackernews_topstory_ids):
+    results = []
+    for item_id in hackernews_topstory_ids:
+        item = requests.get(
+            f"https://hacker-news.firebaseio.com/v0/item/{item_id}.json"
+        ).json()
+        results.append(item)
+
+    return pd.DataFrame(results)
+```
+
+### Building a DAG of assets
+
+Defining multiple assets creates a DAG (directed acyclic graph) of what assets depend on other assets. This DAG, viewable in Dagster's web-based UI, helps you to:
+
+- Understand how your assets relate to each other
+- Empower you and your teammates to act, learn, and debug your pipelines
+
+### Extending with ops and jobs
+
+Asset-based orchestration may seem like a new way to approach data orchestration, but it is similar to traditional task-based orchestration. Under the hood, assets are a wrapper around Dagster's task-based framework. Therefore, all work done with SDAs can also done with tasks. You can even use multiple Ops to create an asset.
+
+Dagster's ideology believes that data engineers should use assets. However, users can use Dagster's Ops and Jobs' task-based framework when assets are not the right fit. Some common situations to use the task-based framework are when you are:
+
+- Sending emails
+- Generating assets you do not know of ahead of time
+
+Once you've completed the basic section of this tutorial, the advanced tutorial explains how to create a complex pipeline that uses both assets and Ops and Jobs.
```

Comment:
"For this tutorial, we're going to focus exclusively on pipelines built using software-defined-assets,  but later  tutorials will explain how to create pipelines that use Assets, Ops and Jobs.

Reply:
I agree with Jarred's suggestion, but suggest a few tweaks:

> For this tutorial, we're going to focus exclusively on pipelines built using Software-defined-assets, but later tutorials will explain how to create pipelines that use Assets, Ops, and Jobs.